From 81a96951349c4de62b6f0451a7799868a37158a0 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
Date: Wed, 8 Jul 2015 20:35:09 +0000
Subject: [PATCH] Fix hoe version conflict

Signed-off-by: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
---
 rubygem-hoe-gemspec.gemspec | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/rubygem-hoe-gemspec.gemspec b/rubygem-hoe-gemspec.gemspec
index 6445527..0385fa6 100644
--- a/rubygem-hoe-gemspec.gemspec
+++ b/rubygem-hoe-gemspec.gemspec
@@ -25,18 +25,15 @@ Gem::Specification.new do |s|
     s.specification_version = 3
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<hoe>, [">= 2.2.0"])
+      s.add_runtime_dependency(%q<hoe>, [">= 2.6.1"])
       s.add_development_dependency(%q<rubyforge>, [">= 2.0.4"])
       s.add_development_dependency(%q<rake>, [">= 0"])
-      s.add_development_dependency(%q<hoe>, [">= 2.6.1"])
     else
-      s.add_dependency(%q<hoe>, [">= 2.2.0"])
       s.add_dependency(%q<rubyforge>, [">= 2.0.4"])
       s.add_dependency(%q<rake>, [">= 0"])
       s.add_dependency(%q<hoe>, [">= 2.6.1"])
     end
   else
-    s.add_dependency(%q<hoe>, [">= 2.2.0"])
     s.add_dependency(%q<rubyforge>, [">= 2.0.4"])
     s.add_dependency(%q<rake>, [">= 0"])
     s.add_dependency(%q<hoe>, [">= 2.6.1"])
-- 
2.4.2

